setup.py

```diff
@@ -153,6 +153,7 @@ def run_tests(self):
     install_requires=install_requires,
     tests_require=tests_require,
     test_suite="nose.collector",
+    data_files=[('', ['LICENSE.txt'])],
```

Comment:
Should have been double quotes. Updated in PR ( https://github.com/nanshe-org/nanshe/pull/447 ).